airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -220,26 +229,29 @@ def create_empty_table(self,
         :type table_id: str
         :param schema_fields: If set, the schema field list as defined here:
         https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.load.schema
-        :param labels: a dictionary containing labels for the table, passed to BigQuery
+        :param labels: a dictionary containing labels for the table,
+        passed to BigQuery
         :type labels: dict
 
         **Example**: ::
 
-            schema_fields=[{"name": "emp_name", "type": "STRING", "mode": "REQUIRED"},
-                           {"name": "salary", "type": "INTEGER", "mode": "NULLABLE"}]
+            schema_fields=[
+            {"name": "emp_name", "type": "STRING", "mode": "REQUIRED"},
+            {"name": "salary", "type": "INTEGER", "mode": "NULLABLE"}
+            ]
 
         :type schema_fields: list
-        :param time_partitioning: configure optional time partitioning fields i.e.
-            partition by field, type and expiration as per API specifications.
+        :param time_partitioning: configure optional time partitioning
+            fields i.e. partition by field, type and expiration
+            as per API specifications.
 
             .. seealso::
             https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#timePartitioning
         :type time_partitioning: dict
 
         :return:
         """
-        if time_partitioning is None:
```

Comment:
Why remove this?

Reply:
logic in DRY, it was placed several times in code, and only one place where was a critical type of local var time_partitioning - in function _cleanse_time_partitioning, so I moved to check inside function, only to avoiding duplicate this check